massgov/mayflower-patternlab
# DP-9017  image placement on guide pages using focal point

Adds a variant for the focal point update to the illustrated header, including adding optional retina display image

## Description
Adds image-centered variant for illustrated header
update illustrated header template for optional retina display image

## Related Issue / Ticket

- [DP-9017](https://jira.mass.gov/browse/DP-9017)


## Steps to Test

1. The image centered variant can be visited at /?p=organisms-illustrated-header-image-centered
2. View that page on a retina and non retina screen to see the placeholder image change 

## Screenshots
On retina:
<img width="1671" alt="screen shot 2018-05-24 at 12 01 39 pm" src="https://user-images.githubusercontent.com/18662734/40500732-4a8e362a-5f4b-11e8-8a9c-f6c6580697e6.png">

On standard:
![screen shot 2018-05-24 at 12 01 52 pm](https://user-images.githubusercontent.com/18662734/40500739-5386c774-5f4b-11e8-99fe-1d215efc7abc.png)


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
